packages/sync-actions/src/utils/copy-empty-array-props.js

```diff
@@ -13,46 +13,41 @@ const CUSTOM = 'custom'
 export default function copyEmptyArrayProps(oldObj = {}, newObj = {}) {
   if (!isNil(oldObj) && !isNil(newObj)) {
     const nextObjectWithEmptyArray = Object.entries(oldObj).reduce(
-      (nextObject, [key, value]) => {
-        const merged = {
-          ...newObj,
-          ...nextObject,
-        }
```

Comment:
This combined object was not needed as it is exactly what the accumulator does in the `reduce`.